### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arispen**
add modify instance attribute method to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -428,4 +428,19 @@ Create a new snapshot of the specified EBS volume.
...
+8. Wait Until Operation End (Boolean) **Optional** - If true wait until the end of the operation. The operation ends when the snapshot is completed.
+
+## Method: Modify Instance Attribute
+**Description**
+
+Modify the instance attribute. This method calls ec2 [ModifyInstanceAttribute](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/EC2.html#modifyInstanceAttribute-property)
+
+**Parameters**
+1. Access Key - This is a parameter taken from the vault to access AWS
+2. Secret Key - This is a paramer taken from the vault to access AWS
```
Comment (kdebowski-kaholo) [inline README.md:440]: typo

File: README.md [hunk 2]
```diff
@@ -428,4 +428,19 @@ Create a new snapshot of the specified EBS volume.
...
+3. Region 
+4. Instance IDs - Instance IDs of all instances you want to modify. Accecpts either a string with one or multiple instance IDs, each seperated with a new line,
```
Comment (kdebowski-kaholo) [inline README.md:442]: "Accecpts" typo

File: README.md [hunk 3]
```diff
@@ -428,4 +428,19 @@ Create a new snapshot of the specified EBS volume.
...
+    or an Array of instance IDs strings passed from code.
+5. attribute - (option) choose the attribute to modify
+6. attributeValue (string) specify the new value of an attrubute
```
Comment (kdebowski-kaholo) [inline README.md:445]: "attrubute" typo

**Review by kdebowski-kaholo — CHANGES_REQUESTED** [review]

**Review by kenskyfish — APPROVED** [review]
Not sure if this doco is all good, but merging only to provide base for further refinement of README.md

**Review by kenskyfish — DISMISSED** [review]
Sure thing

**Review by kdebowski-kaholo — APPROVED** [review]

**Pull request merged**